package.json

```diff
@@ -40,6 +40,7 @@
     "minimist": "~1.2.5",
     "nedb-async": "~0.1.3",
     "ow": "~0.17.0",
+    "sinon": "^14.0.0",
```

Comment:
Are we sure we need this dependency among the `dependencies` instead of the `devDependencies`?